backend/routes/api/resources.js

```diff
@@ -0,0 +1,151 @@
+const express = require("express");
+const router = express.Router();
+const Resource = require("../../models/Resource");
+const { celebrate, Joi } = require("celebrate");
+
+// get all resources
+router.get("/", async (req, res) => {
+  const resources = await Resource.find({});
+  res.json({
+    code: 200,
+    result: resources,
+    success: true
+  });
+});
+
+// get one resource
+router.get("/:resource_id", async (req, res) => {
+  const resourceId = req.params.resource_id;
```

Comment:
Let's add some Celebrate validation here with params. I think we can use some of the API with `Joi.number()` like this (mentioned in [this](https://dev.to/dangolant/hapi-er-express-routing-with-celebrate-and-joi-76f) article):

```
body: Joi.object().keys({
        important_value: Joi.string().required(), // look, type enforcement!
        data1: Joi.number().integer(),
        data2: Joi.string().default('admin') // hey, and defaults!
      }),
      query: {
        token: Joi.string().token().required() // you can use one object to 
                                               // validate body, query, 
                                               // and params all at once
      }
```